easybuild/easyconfigs/i/IPython/IPython-5.8.0-fosscuda-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,231 @@
+easyblock = 'Bundle'
```

Comment:
@akesandgren Please consider updating to `PythonBundle` (which allows a fair amount of cleanup below)